### DISCUSSION TITLE
google-services.json not found

### DISCUSSION THREAD

**Discussion by selfee-jp** [quasarframework/quasar#9410] [CLI - Capacitor / Cordova modes] [answered]
I am using Capacitor.
I'm trying to use the API for push notifications.
When I run "quasar build -m capacitor -T android", the message "google-services.json not found" is displayed.
Where should I put the file "google-services.json"?
Will it be loaded automatically?
I put it in the <project folder>.

**Comment by metalsadman** [comment]
you are probably using a capacitor plugin that requires you to generate that file, if so check that plugin's docs.

**Reply by selfee-jp** [reply]
Yes, I am trying to use "Push notifications".
The manual says to put it in the module (app level) directory.
How about Quasar?
I was able to solve it.
If I put it in "src-capacitor/android/app", it seems to load.

**Comment by selfee-jp** [comment, marked as answer]
It turns out that "google-services.json" should be placed in "src-capacitor/android/app". It seems to be loading "google-services.json" automatically.
It seems that "src-capacitor" will not be rewritten after the build.

If you have upgraded "gradle", should you try to remove it?
